code-of-conduct.md

```diff
@@ -0,0 +1,3 @@
+# Code of Conduct
+
+https://docs.silverstripe.org/en/3.1/contributing/code_of_conduct/
```

Comment:
All the recently upgraded modules have this link, so I would prefer to fix them on a separate thread.